app/views/users/display.html.erb

```diff
@@ -0,0 +1 @@
+<h2>users display<h2>
```

Comment:
- [ ] Kindly rename this file to reflect the action defined in the controller: `show.html.erb`